icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -73,14 +83,25 @@
 #
 # ICU4J_ROOT:    Path to root of ICU4J sources, below which is the main dir.
 #
+# TOOLS_ROOT:    Path to root of ICU tools directory, below which is (e.g.) the
+#                cldr and unicodetools dirs.
+#
 #----
 #
 # If you are adding or removing locales, or specific kinds of locale data,
 # there are some xml files in the ICU sources that need to be updated (these xml
 # files are used in addition to the CLDR files as inputs to the CLDR data build
 # process for ICU):
 #
-#    icu4c/source/data/icu-config.xml - Update <locales> to add or remove
+# The primary file to edit for ICU 67 and later is
+#
+#    $TOOLS_ROOT/cldr/cldr-to-icu/build-icu-data.xml
+#
+# The files used in previous versions are the following; unsure whether these
```

Comment:
They are definitely not used. The new tooling uses nothing other than the new API and code/data in its own package/directory.

Reply:
OK, I will update the comment when I do the next round of CLDR-to-ICU integration for CLDR release-37-alpha2 next week.